### PR TITLE
github actions: fix Docker build globbing

### PR DESCRIPTION
**Pull request by iameli**
Lil fix to the Docker image builds

### CODE REVIEW

File: packages/www/package.json [hunk 1]
```diff
@@ -12,7 +12,7 @@
     "docs-positions": "node ./generate-docs-positions.js",
     "docker:build": "docker build $(printf ' -t livepeerci/www:%s' ${DOCKER_TAGS:-master}) -f Dockerfile ../..",
     "docker:push": "for TAG in ${DOCKER_TAGS:-dev}; do docker push livepeerci/www:${TAG}; done",
-    "pkg": "pkg --out-path=bin --no-bytecode --public-packages \"*\" --public . --compress GZip",
+    "pkg": "pkg --out-path=bin --no-bytecode --public-packages \"*\" --public . --compress GZip ",
```
Comment (victorges) [inline packages/www/package.json:15]: Can revert the 2nd commit i guess

**Review by victorges — APPROVED** [review]
lgtm

File: packages/www/package.json (hunk 1 above)
Comment (iameli) [reply packages/www/package.json:15]: after i get a clean master build i guess

Comment (victorges) [reply packages/www/package.json:15]: o tru

**Pull request merged**